### DISCUSSION TITLE
Sponsoring Rocket development?

### DISCUSSION THREAD

**Discussion by szabgab** [rwf2/Rocket#2786] [Feedback]
I am writing a [series of blog posts](https://dev.to/szabgab/github-sponsors-a-series-on-giving-an-receiving-4fmb) on sponsoring various open source projects via GitHub Sponsors with a focus on Rust developers and Rust projects.

I am going to write about Rocket soon and noticed that at the bottom of the [web site](https://rocket.rs/) there are a few sponsors listed and two links to sponsor, one to [GitHub Sponsors](https://github.com/sponsors/rwf2)  and one to [Open Collective](https://opencollective.com/rwf2/donate). The latter says: "This collective doesn't have a host and can't accept financial contributions".

IMHO something needs to be fixed there.

**Comment by blackghost1987** [comment]
I think this might be related to this sad news: https://opencollective.com/foundation/updates/announcement-we-are-dissolving-open-collective-foundation-at-the-end-of-this-year 😞 
This is quite recent, so I assume things are still in flux. But you're right, the site will have to be updated 👍

**Reply by szabgab** [reply]
Oh. I see there is also a link from here too: https://rwf2.org/

**Reply by SergioBenitez** [reply]
Yes that's exactly right. We posted about it on the collective page, but it seems like that post is now inaccessible. Here's the post:

The Open Collective Foundation (OCF), our fiscal host, recently announced its decision to dissolve. In light of these events, Software in the Public Interest (SPI), a 501(c)(3) public charity and fiscal host, has generously extended an invitation, and we are now actively working on transitioning to SPI as our new fiscal host. We anticipate finalizing the transition by March 25th. In the interim, we are unable to accept donations or utilize existing funds.

As part of the transition, all funds will be transferred to SPI. Once the transition completes, recurring donations will need to be reestablished. We are still finalizing the details for processing and accepting donations through the SPI and will provide clear instructions and next steps as soon we have them.

Lastly, it is unlikely that we will be able to continue operating on the Open Collective platform. Please know that I am personally committed to ensuring that all fiscal matters pertaining to the RWF2 remain transparent and accessible to all, irrespective of the medium. I'll be announcing more on this soon.

In summary:

- The Open Collective Foundation (OCF), our fiscal host, is dissolving.
- Software in the Public Interest (SPI), a 501(c)(3) charity, will be our new fiscal host.
- We aim to complete the transition to SPI by March 25th.
- We cannot currently accept donations or use existing funds.
- All funds will be transferred to SPI.
- Recurring donations will need to be set up again.
- Details on donating through SPI will be shared soon.
- It's unlikely we'll continue on the Open Collective platform.
- We're committed to maintaining transparency for everything fiscal.
- More updates and exciting developments to be shared soon.

Thank you all for your continued support. We've been working behind the scenes on some exciting developments, and I'm looking forward to sharing them with you soon!

Sincerely,
Sergio Benitez

**Reply by SergioBenitez** [reply]
As an update to that: we did finalize the transition to SPI. The process has taken much longer than anticipated, however. Our funds arrived at SPI 4 days ago. We are not yet able to use those funds nor accept new funds that go directly to SPI in a manner that's commensurate with our sponsorship tiers, though we are actively working with SPI to realize that as soon as possible.

As for the exciting bit of news, we have kicked off our uGrant program. The first such grant is here https://github.com/rwf2/rwf2/pull/9, which will be awarded in the next 10 days. Two more grants are following close behind.

**Comment by szabgab** [comment]
[here](https://dev.to/szabgab/sponsoring-the-rust-based-rocket-web-framework-2h39) is my blog post about sponsoring Rocket. Please "like it" and help sharing it.
